docs/src/main/asciidoc/security-oidc-bearer-token-authentication-tutorial.adoc

```diff
@@ -14,6 +14,8 @@ The bearer tokens are issued by OIDC and OAuth 2.0 compliant authorization serve
 
 To better understand OIDC Bearer authentication, see xref:security-oidc-bearer-token-authentication.adoc[OIDC Bearer authentication].
 
+If you want to protect web applications by using OIDC authorization code flow authentication, see xref:security-oidc-code-flow-authentication-concept.adoc[OIDC authorization code flow authentication].
```

Comment:
```suggestion
If you want to protect web applications by using OIDC Authorization Code Flow authentication, see xref:security-oidc-code-flow-authentication-concept.adoc[OIDC authorization code flow authentication].
```

Reply:
I have another local PR for fixing the capitalization of OIDC Authorization Code Flow mechanism, but as this is being touched here, may as well apply it directly here too.

Reply:
Sorry @michelle-purcell that this PR is getting in the way a bit :-), let me apply your suggestions, thanks

Reply:
@sberyozkin - Not at all. I have several atomic PRs open around security right now. Thanks so much.